alerts.go

```diff
@@ -208,6 +214,22 @@ func formatExpressionOneline(expr string) string {
 	return strings.Replace(strings.Replace(expr, "( ", "(", -1), " )", ")", -1)
 }
 
+func formatCheckMessage(msg string) string {
+	truncated := false
+	if index := strings.IndexAny(msg, "\n\r"); index != -1 {
+		msg = msg[0:index]
+		truncated = true
+	}
+	if msgU := utf8string.NewString(msg); msgU.RuneCount() > 100 {
+		msg = msgU.Slice(0, 100)
```

Comment:
What's the benefit of using utf8string package, rather than casting to `[]rune`?

Reply:
Just I forgot about `[]rune` 😱 . Fixed to use `[]rune`.